xen/include/public/io/displif.h

```diff
@@ -202,6 +202,9 @@
  *      Width and height of the connector in pixels separated by
  *      XENDISPL_RESOLUTION_SEPARATOR. This defines visible area of the
  *      display.
+ *      If backend provides extended display identification data (EDID) with
+ *      XENDISPL_OP_GET_EDID request then EDID values must take precedence
+ *      over the resolutions defined here.
  *
```

Comment:
What to do if for some guest we need exactly specific resolution, and we describe it inside domX.cfg but we have hardware monitor that reports EDID with other config, not same as we describe. Is there any way to:
- force usage of settings from domain config
- disable reading of EDID
Something like "force" in config or "ignore-edid" in driver.
Also this need to be described in all domain configs.

Reply:
These are all up to backend implementation. E.g. backend may decide not to provide EDID at all and then front will use XenStore config. BTW, current backend implementation uses XenStore values (config) to construct the relevant EDID structure.